src/frontend/tsx/Games.tsx

```diff
@@ -32,7 +32,7 @@ export function Games () {
                 key={key}
                 className={`game active ${colorClass} ${status}`}
                 title={gameID}
-                onClick={() => setDisplayGame(activeGame)}>
+                onClick={() => { setDisplayGame(activeGame); setDisplayIndex(activeGame.info.moves.length) }}>
```

Comment:
Array indices are 0-based, so `activeGame.info.moves.length` is actually one past the end of the Array